pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -851,6 +851,9 @@ def parse_args_and_make_package(args=None):
     ap.add_argument('--extra-manifest-xml', default='',
                     help=('Extra xml to write directly inside the <manifest> element of'
                           'AndroidManifest.xml'))
+    ap.add_argument('--extra-manifest-application-entry', default='',
```

Comment:
Can we rename `--extra-manifest-application-entry` to `--extra-manifest-application-xml`, since it adds some extra XML into <manifest><application>?

Reply:
Yes we can. 

> since it adds some extra XML into <manifest><application>?

But I add section XML inside <application> </application>. This section can include any elements that can need inside.